### DISCUSSION TITLE
Testing, is  pipenv install --dev no longer required?

### DISCUSSION THREAD

**Discussion by River-Mochi** [builder555/PineSAM#164] [Q&A] [answered]
#### 1. Old instructions 
![image](https://user-images.githubusercontent.com/97197236/230074950-af141aa1-fd60-4f9e-9994-8226a3253a64.png)

#### 2. New instructions 
does not say it needs to run in backend folder or to install --dev
![image](https://user-images.githubusercontent.com/97197236/230074023-73178c99-0f7a-42ec-83a2-7ce7e38beed4.png)

**Comment by builder555** [comment, marked as answer]
`--dev` is required for running unit tests, but it's done inside setup-dev.sh, so users don't have to run it explicitly.
